docs/builders/xenserver-iso.html.markdown

```diff
@@ -150,13 +150,9 @@ each category, the available options are alphabetized and described.
 ```
 
 * `shutdown_command` (string) - The command to use to gracefully shut down
-  the machine once all the provisioning is done. By default this is an empty
-  string, which tells Packer to just forcefully shut down the machine.
-
-* `shutdown_timeout` (string) - The amount of time to wait after executing
```

Comment:
The `shutdown_timeout` is actually hard coded [here](https://github.com/ddelnano/packer-plugin-xenserver/blob/e4b9d11d853c64a001c655f5cfebca8cf1a668ca/builder/xenserver/common/step_shutdown.go#L47) so I'm removing this as well.